WDL2CWL_example/GATK4/MarkDuplicates.cwl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env cwl-runner
```

Comment:
Did https://github.com/common-workflow-library/bio-cwl-tools/blob/release/GATK/GATK-MarkDuplicates.cwl not work for you?

Reply:
ngsap2@ubuntu:~$ sudo docker run -v /var/run/docker.sock:/var/run/docker.sock -v "$PWD":"$PWD" -w="$PWD" commonworkflowlanguage/cwltool Ambarish_Kumar_SOP-GATK-SAR-CoV-2.cwl Ambarish_Kumar_SOP-GATK-SAR-CoV-2_inputs.yml
INFO /usr/local/bin/cwltool 2.0.20200107113851
INFO Resolved 'Ambarish_Kumar_SOP-GATK-SAR-CoV-2.cwl' to 'file:///home/ngsap2/Ambarish_Kumar_SOP-GATK-SAR-CoV-2.cwl'
tools/picard/picard_MarkDuplicates.cwl:24:3: object id `tools/picard/picard_MarkDuplicates.cwl#alignments` previously defined
ERROR Tool definition failed validation:
Ambarish_Kumar_SOP-GATK-SAR-CoV-2.cwl:83:12:   Workflow step output 'bam_duprem' does not correspond
                                              to
tools/picard/picard_MarkDuplicates.cwl:101:1:   tool output (expected 'alignments', 'log',
                                                'metrics')
Ambarish_Kumar_SOP-GATK-SAR-CoV-2.cwl:81:5:     Step is missing required parameters 'alignments',
                                              'alignments_are_sorted', 'remove_duplicates'